.ci/docker-compose-file/scripts/run-emqx.sh

```diff
@@ -19,17 +19,27 @@ fi
 } >> .ci/docker-compose-file/conf.cluster.env
 
 is_node_up() {
-  local node
-  node="$1"
-  docker exec -i "$node" \
-         bash -c "emqx eval \"['emqx@node1.emqx.io','emqx@node2.emqx.io'] = maps:get(running_nodes, ekka_cluster:info()).\"" > /dev/null 2>&1
+  local node="$1"
+  if [ "${IS_ELIXIR:-no}" = "yes" ]
+  then
+    docker exec -i "$node" \
+           bash -c "emqx eval \"[:\\\"emqx@node1.emqx.io\\\", :\\\"emqx@node2.emqx.io\\\"] = :ekka_cluster.info()[:running_nodes]\""
+  else
+    docker exec -i "$node" \
+           bash -c "emqx eval \"['emqx@node1.emqx.io','emqx@node2.emqx.io'] = maps:get(running_nodes, ekka_cluster:info()).\"" > /dev/null 2>&1
+  fi
```

Comment:
this is getting complex.
maybe add (in another PR) a `emqx_ctl` command to print the clustered nodes.
5.0 command line outputs are subject to refactor to (pretty) print JSON.

Reply:
I noticed that there's `emqx ctl cluster status` that more or less prints the result of `ekka_cluster:info/0`, but outputs in Erlang syntax.  If we had a variation formatted in JSON, it could be easier to massage the results in the shell using `jq` and `grep`.

Maybe we could try to add a `--json` flag to `emqx ctl cluster status` to format the output in JSON and then use that to assert the cluster (in another PR)?

Reply:
we'll go with only JSON.
formatting JSON to tabular will be easy.